tests/unit/test-firefox/test.firefox.js

```diff
@@ -263,6 +294,113 @@ describe('firefox', () => {
 
   });
 
+  describe('isDefaultProfile', () => {
+
+    it('detects common Firefox default profiles specified by name',
+       async () => {
+         const isDefault = await firefox.isDefaultProfile('default');
+         assert.equal(isDefault, true);
+
+         const isDevEditionDefault = await firefox.isDefaultProfile(
+           'dev-edition-default'
+         );
+         assert.equal(isDevEditionDefault, true);
+       });
+
+    it('allows profile name if is not listed as default in profiles.ini',
+      async () => {
+        return withTempDir(async (tmpDir) => {
+          const profilesDirPath = tmpDir.path();
+          const FakeProfileFinder = createFakeProfileFinder(profilesDirPath);
+
+          await createFakeProfilesIni(profilesDirPath, [
+            {
+              Name: 'manually-set-default',
+              Path: 'fake-default-profile',
+              IsRelative: 1,
+              Default: 1,
+            },
+          ]);
+
+          const isDefault = await firefox.isDefaultProfile(
+            'manually-set-default', FakeProfileFinder
+          );
+          const isNotDefault = await firefox.isDefaultProfile(
+            'unkown-profile-name', FakeProfileFinder
+          );
+
+          assert.equal(isDefault, true);
+          assert.equal(isNotDefault, false);
+        });
+      });
+
+    it('allows profile path if is not listed as default in profiles.ini',
+       async () => {
+         return withTempDir(async (tmpDir) => {
+           const profilesDirPath = tmpDir.path();
+           const FakeProfileFinder = createFakeProfileFinder(profilesDirPath);
+
+           await createFakeProfilesIni(profilesDirPath, [
+             {
+               Name: 'default',
+               Path: 'fake-default-profile',
+               IsRelative: 1,
+             },
+             {
+               Name: 'dev-edition-default',
+               Path: 'fake-devedition-default-profile',
+               IsRelative: 1,
+             },
+             {
+               Name: 'manually-set-default',
+               Path: '/tmp/fake-manually-default-profile',
+               Default: 1,
+             },
+           ]);
+
+           const isFirefoxDefaultPath = await firefox.isDefaultProfile(
+             path.join(profilesDirPath, 'fake-default-profile'),
+             FakeProfileFinder
+           );
+           assert.equal(isFirefoxDefaultPath, true);
+
+           const isDevEditionDefaultPath = await firefox.isDefaultProfile(
+             path.join(profilesDirPath, 'fake-devedition-default-profile'),
+             FakeProfileFinder
+           );
+           assert.equal(isDevEditionDefaultPath, true);
+
+           const isManuallyDefault = await firefox.isDefaultProfile(
+             '/tmp/fake-manually-default-profile',
+             FakeProfileFinder
+           );
+           assert.equal(isManuallyDefault, true);
+
+           const isNotDefault = await firefox.isDefaultProfile(
```

Comment:
This case looks like it's covered up above. Is there something different about testing it again here?